src/Dialect/ONNX/Transforms/Decompose.cpp

```diff
@@ -685,6 +684,20 @@ bool hasDefaultDilation(ArrayAttr dilation) {
   SmallVector<int64_t, 3> vDilation = getIntVectorFromArrayAttr(dilation);
   return llvm::all_of(vDilation, [](int64_t d) { return d == 1; });
 }
+/*
```

Comment:
I think we are mostly using it and not block comment.
```suggestion
//
```

Reply:
Done